lib/frameworks/mocha.js

```diff
@@ -17,6 +12,11 @@ module.exports.run = function(config, specs, capabilities) {
         requires = config.mochaOpts.require,
         runner;
 
+    /**
+     * enable generators if supported
+     */
+    coMocha(Mocha);
```

Comment:
One thing about this is that we now have a mochaCo dependency even though the user doesn't necessarily uses Mocha (same for Jasmine). I am a bit afraid that we introduce unnecessary dependency overhead only to get the code a bit shorter